src/app/components/Curation/index.tsx

```diff
@@ -91,6 +95,8 @@ export default ({
       );
     case RADIO_SCHEDULE:
       return <RadioSchedule initialData={radioSchedule} />;
+    case EMBED:
+      return vjEmbed ? <Embed oembed={embed} type="" /> : null;
```

Comment:
Do we need ensure that there is an embed before we can render the component? Also, is type a mandatory prop here? If so, is there another value other than `""` that we can use?

```suggestion
      return <Embed oembed={embed} type="" />;
```

Reply:
Oooh yes I had forgotten to ask about the "", I was unsure what value to insert at the time but it is a mandatory prop so I'd popped in the empty string as a temp workaround in order to check my code was working 😅 so I'll have to change this still, thank you for the reminder.

I originally had `return <Embed oembed={embed} type="" />` but it cause oembed to scream about a type error I was unable to solve at the time, so I can revisit this if we'd rather not have the check in there first.

Reply:
It's OK if we still need the check - especially if TS was screaming at you! 😱 And if an empty type is OK, then let's leave that too 😸